### PR TITLE
Split the "work pools and workers" page into two pages

### PR DESCRIPTION
**Pull request by djsauble**
Work pools: https://prefect-bd373955-split_the_deploy_work_pools_and_workers_in.mintlify.app/3.0rc/deploy/control-runs/work-pools
Workers: https://prefect-bd373955-split_the_deploy_work_pools_and_workers_in.mintlify.app/3.0rc/deploy/control-runs/workers

For more context see: https://github.com/PrefectHQ/prefect/issues/14094

### Example

Most links to this page reference either work pools or workers, not both. This is a smell which indicates that this page should be two, not one. It's also a very long page, so splitting it up makes it a bit more manageable.

### Checklist

- [x] This pull request includes a label categorizing the change e.g. `maintenance`, `fix`, `feature`, `enhancement`, `docs`.
- [x] This pull request references any related issue by including "closes `<link to issue>`"
  
  - If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] If this pull request adds new functionality, it includes unit tests that cover the changes
- [x] If this pull request removes docs files, it includes redirect settings in `mint.json`.
- [x] If this pull request adds functions or classes, it includes helpful docstrings.

### CODE REVIEW

File: docs/3.0rc/deploy/control-runs/prefect-deploy.mdx [hunk 1]
```diff
@@ -4,6 +4,9 @@ description: Learn how to schedule and trigger flow runs through the API and eas
 ---
 
 In this guide, you will configure a deployment that uses a work pool for dynamically provisioned infrastructure.
+Work pools and workers bridge the Prefect _orchestration environment_ with your _execution environment_. 
+When a [deployment](/3.0rc/deploy/serve-flows/) creates a flow run, it is submitted to a specific work pool for scheduling. 
+A worker running in the execution environment polls its respective work pool for new runs to execute; or the work pool submits flow runs to serverless infrastructure directly, depending on your configuration.
```
Comment (discdiver) [inline docs/3.0rc/deploy/control-runs/prefect-deploy.mdx:9]:
```suggestion
A worker running in the execution environment polls its respective work pool for new runs to execute; or the work pool submits flow runs to serverless infrastructure directly, depending on the type of work pool.
```

File: docs/3.0rc/deploy/control-runs/workers.mdx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+Workers have two statuses: `ONLINE` and `OFFLINE`. A worker is online if it sends regular heartbeat messages to the Prefect API. 
+If a worker misses three heartbeats, it is considered offline. By default, a worker is considered offline a maximum of 90 seconds 
+after it stopped sending heartbeats, but you can configure the threshold with the `PREFECT_WORKER_HEARTBEAT_SECONDS` setting.
+
+### Start a worker
+
+Use the `prefect worker start` CLI command to start a worker. You must pass at least the work pool name. 
+If the work pool does not exist, it will be created if the `--type` flag is used.
+<div class="terminal">
```
Comment (discdiver) [inline docs/3.0rc/deploy/control-runs/workers.mdx:74]:
```suggestion
```

Comment (discdiver) [reply docs/3.0rc/deploy/control-runs/workers.mdx:74]: Let's remove the <div class=terminal>s and closing </div> that doesn't do anything any more.

File: docs/3.0rc/deploy/control-runs/workers.mdx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+```bash
+prefect worker start -p [work pool name]
+```
+</div>
```
Comment (discdiver) [inline docs/3.0rc/deploy/control-runs/workers.mdx:78]:
```suggestion
```

**Review by discdiver — COMMENTED** [review]
Thank you. Minor suggestions.

**Review by billpalombi — APPROVED** [review]
I would like to make further edits to the workers page but we can treat that as a separate PR

**Pull request merged**